contrib/cf-remote/cf_remote/remote.py

```diff
@@ -237,6 +237,8 @@ def install_host(
         if edition == "enterprise":
             tags.append("hub" if hub else "agent")
         tags.append("64" if data["arch"] in ["x86_64", "amd64"] else data["arch"])
+        if data["arch"] in ["i386", "i486", "i586", "i686"]:
```

Comment:
What about `x86`? I think we use just that in some place too.

Reply:
You mean, if `uname -m` in get_info() got `x86`, set the 32 bit tag? Is it a possible value?

https://gcc.gnu.org/backends.html
https://gcc.gnu.org/onlinedocs/gcc-4.5.3/gcc/i386-and-x86_002d64-Options.html

Don't mention plain x86 or even specifically x86_64, but do mention i[3-6]86.

Reply:
I didn't know where `data["arch"]` was coming from, I thought it was from our data somewhere.